lls/main.c

```diff
@@ -337,6 +394,15 @@ lls_proc(void *arg)
 			 * happen. In fact, we may want to reduce the amount
 			 * of times this is called, since reading the HPET
 			 * timer is inefficient. See the timer application.
+			 *
+			 * Also send gratuitous ARP/ND messages to fulfill
+			 * the LACP requirement.
+			 *
+			 * XXX The LACP requirement could be starved if
+			 * the LLS block receives a lot of requests but
+			 * we are unable to answer them -- i.e. the
+			 * number of requests > 0 for a sustained
+			 * period but we never invoke the TX burst.
```

Comment:
I modified `process_pkts()` to do this, but didn't touch the timer during a the cache scan since it involved changing a lot of function prototypes and is a rare opportunity to skip a cycle anyway.